packages/fuel-indexer/src/service.rs

```diff
@@ -286,3 +298,19 @@ async fn create_service_task(
         }
     }
 }
+
+async fn get_start_block(conn: &mut IndexerConnection, manifest: &Manifest) -> u64 {
+    let resumable = manifest.resumable.unwrap_or(false);
+    if resumable {
+        let start_block = queries::last_block_height_for_indexer(
+            conn,
+            &manifest.namespace,
+            &manifest.identifier,
+        )
+        .await
+        .unwrap_or(0) as u64;
+        start_block
+    } else {
+        manifest.start_block.unwrap()
```

Comment:
This will cause the service to panic and crash if the manifest does not have a number for the `start_block` field. Up to now, we've allowed the manifest have the field undefined or assigned a dummy value of `~` by convention.